contracts/token/psp1155/src/traits.rs

```diff
@@ -249,60 +261,52 @@ pub trait IPSP1155: PSP1155Storage {
         self.get().balances.get(&(id, owner)).cloned().unwrap_or(0)
     }
 
-    fn _is_approved_for_all(&self, _account: AccountId, _operator: AccountId) -> bool {
+    fn _is_approved_for_all(&self, account: AccountId, operator: AccountId) -> bool {
         self.get()
             .operator_approval
-            .get(&(_account, _operator))
+            .get(&(account, operator))
             .cloned()
             .unwrap_or(false)
     }
 
     fn _increase_receiver_balance(&mut self, to: AccountId, id: Id, amount: Balance) {
-        let to_balance = self.get_mut().balances.entry((id, to)).or_insert(0);
-        match to_balance.checked_add(amount) {
-            Some(new_to_balance) => *to_balance = new_to_balance,
-            _ => panic!("{}", PSP1155Error::MaxBalance.as_ref()),
-        }
+        self.get_mut()
+            .balances
+            .entry((id, to))
+            .and_modify(|b| *b += amount)
+            .or_insert(amount);
     }
 
     fn _decrease_sender_balance(&mut self, from: AccountId, id: Id, amount: Balance) {
```

Comment:
I think you can remove this function now and use directly `and_modify`

Reply:
It is used 2 times so cleaner to keep it.